### DISCUSSION TITLE
Mouse tooltip suggestions

### DISCUSSION THREAD

**Discussion by CaraPis** [runelite/runelite#15045] [Ideas]
- Add ability to hide mouse tooltips for items you have hiden ground ground items plugin?
- Add ability to toggle background from the mouse tooltip box to a solid colour instead of a transparent theme. I found it sometimes hard to see what's written inside of the tooltip box.

**Comment by MarkPuchalaII** [comment]
I believe the tooltip background has a solution, now.
I went to Settings > Runelite > Overlay Settings > Overlay Color and set the opacity to 100%.
Does that sound like a good fix?
